.github/workflows/fork-ci.yaml

```diff
@@ -14,24 +14,24 @@ jobs:
       REGISTRY_INTERNAL: nvcr.io/nvstaging/mellanox
       REGISTRY_PUBLIC: nvcr.io/nvidia/cloud-native
     steps:
-    - uses: actions/checkout@v4
-      with:
-        sparse-checkout: .
-    - if: github.ref_type == 'branch'
-      name: Determine docker registry and tag (when git branch)
-      run: |
-        echo DOCKER_REGISTRY=$REGISTRY_INTERNAL >> $GITHUB_ENV
-        echo DOCKER_TAG=$(git rev-parse --short HEAD) >> $GITHUB_ENV  # short git commit hash
-    - if: github.ref_type == 'tag'
-      name: Determine docker registry and tag (when git tag)
-      run: |
-        echo DOCKER_REGISTRY=$(echo ${{ github.ref_name }} | sed 's/network-operator-//' | grep -q '-' && echo $REGISTRY_INTERNAL || echo $REGISTRY_PUBLIC) >> $GITHUB_ENV  # use public registry only when release tag has no '-beta*' or '-rc*' suffix
-        echo DOCKER_TAG=${{ github.ref_name }} >> $GITHUB_ENV
-    - name: Store docker registry and tag for following jobs
-      id: store-docker-registry-and-tag
-      run: |
-        echo DOCKER_REGISTRY=$DOCKER_REGISTRY >> $GITHUB_OUTPUT
-        echo DOCKER_TAG=$DOCKER_TAG >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .
+      - if: github.ref_type == 'branch'
+        name: Determine docker registry and tag (when git branch)
+        run: |
+          echo DOCKER_REGISTRY=$REGISTRY_INTERNAL >> $GITHUB_ENV
+          echo DOCKER_TAG=$(git rev-parse --short HEAD) >> $GITHUB_ENV  # short git commit hash
+      - if: github.ref_type == 'tag'
+        name: Determine docker registry and tag (when git tag)
+        run: |
+          echo DOCKER_REGISTRY=$(echo ${{ github.ref_name }} | sed 's/network-operator-//' | grep -q '-' && echo $REGISTRY_INTERNAL || echo $REGISTRY_PUBLIC) >> $GITHUB_ENV  # use public registry only when release tag has no '-beta*' or '-rc*' suffix
+          echo DOCKER_TAG=${{ github.ref_name }} >> $GITHUB_ENV
+      - name: Store docker registry and tag for following jobs
+        id: store-docker-registry-and-tag
+        run: |
+          echo DOCKER_REGISTRY=$DOCKER_REGISTRY >> $GITHUB_OUTPUT
+          echo DOCKER_TAG=$DOCKER_TAG >> $GITHUB_OUTPUT
```

Comment:
All these indentation changes make PR diff hard to read, and make code inconsistent... I'd prefer to format the YAML arrays like
```yaml
this:
- one
- two
instead of:
  - three
  - four
```

Reply:
It's aligned with other/upstream workflow definitions